File: striver/S2L1/bubble.py
size = int(input())
arr = [0]*size
for i in range(0,size):
    arr[i]=int(input())
def bubble_sort(arr):
    for i in range(0,size-1):
        for j in range(0,size-1-i):
            if arr[j] > arr[j+1]:
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            for i in range(0,size):
                print(arr[i],end=" ")
            print()
bubble_sort(arr)
for i in range(0,size):
        print(arr[i],end=" ")
            

File: striver/S2L1/bubble.java
package S2L1;

import java.util.Scanner;

public class bubble {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] arr = new int[n];
        for(int i=0;i<n;i++)
            arr[i] = sc.nextInt();
        bs(arr);
        for(int i=0;i<n;i++)
            System.out.print(arr[i]+" ");
    }
    public static void bs(int[] arr){
        int size = arr.length;
        for(int i=0;i<size-1;i++){
            for(int j=0;j<size-i-1;j++){
                if(arr[j]>=arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
    }
}
